docs/accounts.md

```diff
@@ -0,0 +1,21 @@
+# Blacklight Login Scenarios 
+
+## Unauthenticated User from main menu
+
+## Library Account - From the User Menu Bar
+    * with Netid - devise user type "cas"
+    * with your Alma active barcode -credentials are Last Name and active barcode  devise user type "barcode" - this is a legacy option, no new accounts being set up in this fashion. 
```

Comment:
This 'with your Alma active barcode -credentials are Last Name and active barcode ', does not exist as an option when I click On Your account' -> 'Library account'